app/javascript/Pages/Notifications.jsx

```diff
@@ -1,35 +1,168 @@
-import Layout from "./Layout";
+import axios from "axios";
+import { Inertia } from "@inertiajs/inertia";
+import { IoIosArrowBack } from "react-icons/io";
+// import { Head, Link } from "@inertiajs/inertia-react";
+import { useState } from "react";
 
 export default function Welcome(props) {
-  const Notifications_list = (props) => {
-    console.log(props.notifications);
-    const notificationsList = props.notifications.map((n, index) => (
-      <li key={index}>
-        id: {n.id}
-        text: {n.text}
-        notification_type: {n.notification_type}
-      </li>
-    ));
-
-    return <ul>{notificationsList}</ul>;
+  const [notifications, setNotifications] = useState(props.notifications);
+
+  const requestOnClick = (match_id, answer) => {
+    const url = `/api/match_request/${match_id}`;
+    axios
+      .put(url, {
+        status: answer,
+        headers: window.defaultHeaders,
+      })
+      .then(() => {
+        //update notifications
+        axios
+          .get("/api/notifications")
+          .then((response) => {
+            setNotifications(response.data.notifications);
+          })
+          .catch((error) => {
+            alert(error.response.data.error);
+          });
+      })
+      .catch((error) => alert(error.response.data.error));
+  };
+
+  const reviewOnClick = () => {
+    alert("Review connection!");
+  };
+
+  const avatar = (image) => {
+    return `${image}?tr=,r-max`;
   };
 
   return (
-    <Layout current_user={props.current_user}>
-      <Head title="Welcome" />
-      <h1>Notifications</h1>
-      <p>Unread notifications</p>
-      <Notifications_list notifications={props.notifications.unread} />
-      <p>Read notifications</p>
-      <Notifications_list notifications={props.notifications.read} />
-
-      <Link href="/home" headers={Window.defaultHeaders} as="button">
-        Home
-      </Link>
-    </Layout>
+    <main className="notifications">
+      <div className="notifications-header">
+        <IoIosArrowBack
+          onClick={() => {
+            Inertia.get("/home");
+          }}
+        />
+        <h1 className="notifications-title">Activity</h1>
+      </div>
+
+      <div className="notifications-requests">
+        <h2 className="notifications-requests-title">Classes requests</h2>
+        {notifications.requests.map((notification, index) => {
+          console.log(notification.match.student);
+          return (
+            <div key={index}>
+              <div className="notifications-requests-profile-info">
+                <img
+                  src={avatar(notification.match.student.avatar.url)}
+                  className="notifications-requests-profile-info-avatar"
+                />
+                <p className="notifications-requests-profile-info-text">
+                  <span className="bold">
+                    {notification.match.student.username}
+                  </span>
+                  &nbsp;
```

Comment:
Again, you shouldn't do this.

In this case, you already got a span next to element, so use the span to get that space you want